test/instabug_test.dart

```diff
@@ -257,13 +259,48 @@ void main() {
   });
 
   test('[clearAllExperiments] should call host method', () async {
+    // ignore: deprecated_member_use_from_same_package
     await Instabug.clearAllExperiments();
 
     verify(
       mHost.clearAllExperiments(),
     ).called(1);
   });
 
+  test('[addFeatureFlags] should call host method', () async {
+    await Instabug.addFeatureFlags(
+      List.of([
+        FeatureFlag(name: 'name1', variant: 'variant1'),
+        FeatureFlag(name: 'name2', variant: 'variant2'),
+      ]),
+    );
```

Comment:
Why do we need `List.of` here? Couldn't we just pass the list of feature flags directly to `addFeatureFlags`? 👀
```suggestion
    await Instabug.addFeatureFlags([
      FeatureFlag(name: 'name1', variant: 'variant1'),
      FeatureFlag(name: 'name2', variant: 'variant2'),
    ]);
```